### PR TITLE
Update en strings.xml and README.md

### PR DESCRIPTION
**Pull request by TA874**

### CODE REVIEW

File: values/strings.xml [hunk 1]
```diff
@@ -552,7 +552,7 @@
     <string name="influence_education_low_text">Schools provide low education.</string>
     <string name="influence_education_high_text">High education is provided by high schools.</string>
     <string name="influence_firedep_text">Influence of fire departmens reduces the probabilty of fires.</string>
-    <string name="influence_health_text">Doctors help to keep your inhabitants healthy.</string>
+    <string name="influence_health_text">Doctors and nurses help to keep your inhabitants healthy.</string>
```
Comment (JustAnyones) [inline values/strings.xml:555]: Should remain as is.

**Review by JustAnyones — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ You have just to translate the part in between (here "My Translation"). Special
 
 ### Languages ###
 Every directory contains a `values.xml` file. The corresponding languages are:
+- values-af: Afrikaans
```
Comment (JustAnyones) [inline README.md:14]: Outside the scope of this commit.

File: values/strings.xml [hunk 3]
```diff
@@ -1960,7 +1960,7 @@ Cons: Less challenges</string>
     <string name="specialoffer_cmdbuy">Buy for %s</string>
     <string name="influence_body_disposal_title">Funeral</string>
     <string name="influence_body_disposal_text">Rest in peace.</string>
-    <string name="influence_traffic_title">Traffic</string>
+    <string name="influence_traffic_title">Traffic environment</string>
```
Comment (JustAnyones) [inline values/strings.xml:1963]: Doesn't make sense, should be kept as originally intended.

**Comment by TA874** [comment]
https://github.com/TheoTown-Team/translations/pull/691
https://github.com/TheoTown-Team/translations/pull/692

**Pull request closed without merging**